### PR TITLE
Player 4691

### PR DESCRIPTION
**Pull request by sndy35**
Changes for this PR are already reviewed under https://github.com/ooyala/android-sample-apps/pull/380. Created new branch with jira number PLAYER-4691

### CODE REVIEW

**Review by isheftel-ooyala — APPROVED** [review]
LGTM

**Review by SergeyBicarte — APPROVED** [review]
LGTM

File: OoyalaSkinSampleApp/app/src/main/res/layout/embed_pcode_layout.xml [hunk 1]
```diff
@@ -13,14 +13,14 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_margin="20dp"
+            android:layout_margin="@dimen/activity_layout_margin"
```
Comment (bychkovdmitry) [inline OoyalaSkinSampleApp/app/src/main/res/layout/embed_pcode_layout.xml:16]: resource name "activity_layout_weight" is misleading

File: OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/GeoBlockingActivity.java [hunk 2]
```diff
@@ -46,9 +47,31 @@ void completePlayerSetup(boolean asked) {
...
+      Options.Builder optionsBuilder = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true);
+      if(!selectedFormat.equalsIgnoreCase("default")) {
+        optionsBuilder.setPlayerInfo(new CustomPlayerInfo(selectedFormat)).build();
+      }
+      Options options = optionsBuilder.build();
+
+        //Uncomment next line if you work on STAGING
+        //OoyalaPlayer.setEnvironment(Environment.EnvironmentType.STAGING);
+
+        if(getIntent().getExtras().getString("className").contains("AddAssetActivity")) {
```
Comment (bychkovdmitry) [inline OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/GeoBlockingActivity.java:59]: not reliable, should be at least
```java
getIntent().getExtras().getString("className").contains(AddAssetActivity.class.getSimpleName())
```

**Review by bychkovdmitry — CHANGES_REQUESTED** [review]

**Pull request merged**